addons/im_livechat/report/im_livechat_report_channel_views.xml

```diff
@@ -33,25 +62,26 @@
             <field name="arch" type="xml">
                 <search string="Search report">
                     <field name="channel_name"/>
-                    <field name="partner_id"/>
-                    <filter name="last_24h" string="Last 24h" domain="[('start_date','&gt;', (context_today() - datetime.timedelta(days=1)).strftime('%Y-%m-%d') )]"/>
-                    <filter name="start_date_filter" string="This Week" domain="[
-                        ('start_date', '>=', (datetime.datetime.combine(context_today() + relativedelta(weeks=-1,days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S')),
-                        ('start_date', '&lt;', (datetime.datetime.combine(context_today() + relativedelta(days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S'))]"/>
-                    <filter name="last_30_days" string="Last 30 Days" domain="[
-                        ('start_date', '>=', (datetime.datetime.combine(context_today() + relativedelta(months=-1,days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S')),
-                        ('start_date', '&lt;', (datetime.datetime.combine(context_today() + relativedelta(days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S'))]"/>
-                    <filter name="this_year" string="This Year" domain="[
-                        ('start_date', '>=', (datetime.datetime.combine(context_today() + relativedelta(years=-1,days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S')),
-                        ('start_date', '&lt;', (datetime.datetime.combine(context_today() + relativedelta(days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S'))]"/>
+                    <field name="partner_id" string="Agent"/>
+                    <field name="channel_name" string="Live Chat Channel"/>
```

Comment:
I think the spec asks to change the group by value, not the string. (Keep channel as string, but group by LC channel).